packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -438,7 +438,8 @@ export class BasePickerListBelow<T, P extends IBasePickerProps<T>> extends BaseP
         { this.renderSuggestions() }
         <FocusZone ref={ this._resolveRef('focusZone') }
           className='ms-BasePicker-selectedItems'
-          isInnerZoneKeystroke={ this._isFocusZoneInnerKeystroke } >
+          isInnerZoneKeystroke={ this._isFocusZoneInnerKeystroke }
+          direction={ FocusZoneDirection.horizontal } >
```

Comment:
David mentioned that we really shouldn't be setting this to be just horizontal. If we need to fix alignment or something like that then we should do that.